build/build-changelog.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$UPDATE = " - Added support for latest API updates.";
+$path = __DIR__ . '/../CHANGELOG.md';
+
+$diff = shell_exec("git status --porcelain");
```

Comment:
It looks like `--porcelain` is only supported in versions `2.10.0` and above. This may be an issue with the git versions we use. May want to check on that.